CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+- The OTLP exporter now has two new convenience functions, `NewExportPipeline` and `InstallNewPipeline`, setup and install the exporter in tracing and metrics pipelines. (#1348)
+
```

Comment:
This should be under the `Added` subsection of the `Unreleased` section as new functionality will be added by this PR.  The identifier should refer to the PR that introduced the change.

```suggestion
- The OTLP exporter now has two new convenience functions, `NewExportPipeline` and `InstallNewPipeline`, setup and install the exporter in tracing and metrics pipelines. (#1373)

```

Reply:
Please also note the comment regarding the placement of this item within the `CHANGELOG`.  It needs to go in the appropriate section.